client/blocks/post-share/index.jsx

```diff
@@ -391,7 +391,7 @@ class PostShare extends Component {
 			return (
 				<Notice status="is-success" onDismissClick={ this.dismiss }>
 					{ translate( "We'll share your post on %s.", {
-						args: this.props.scheduledAt.format( 'ddd, MMMM Do YYYY, h:mm:ss a' ),
+						args: this.props.scheduledAt.format( 'LLLL' ),
```

Comment:
This makes the date a little less concrete, because `LLLL` doesn't contain the seconds. Do we want to make it something like `LL, LTS` instead (though that will make it lack the day of the week)?

Reply:
That one is ok, because you can't actually schedule a share with seconds - the UI only takes hours/minutes
<img width="256" alt="screen shot 2018-01-09 at 14 46 51" src="https://user-images.githubusercontent.com/844866/34721617-0f4732d8-f54c-11e7-9fc2-794e762ecf26.png">